modules/secret/README.md

```diff
@@ -0,0 +1,99 @@
+# Terraform Google Secret Manager Secret Submodule
+
+This module makes it easy to create a single Google Secret Manager secret. It supports using custom KMS keys (CMEK) for encrypting the secrets.
+It also supports secret rotation and can configure the rotation notification to be sent to given pubsub topics.
+
+## Usage
+
+Basic usage of this module is as follows:
+
+```hcl
+module "secret-manager" {
+  source  = "GoogleCloudPlatform/secret-manager/google//modules/secret"
```

Comment:
For consistency with some of our other single variants let's call it simple-secret here and throughout.
```suggestion
  source  = "GoogleCloudPlatform/secret-manager/google//modules/simple-secret"
```

Reply:
OK, renamed it to `simple-secret` 👍 .